### ISSUE TITLE
[BUG] Event Link Fields

### ISSUE THREAD

**Issue by vburgett** [rokwire/illinois-app#3580]
When creating an event in the app, the two fields for entering a link should be consistent. 

**Online url** and **Add Event Website Link** 
These two fields only create a tappable link/button if and when a user knows to include "https://" or "http://" 

**Online url field**
Add "Confirm URL"

**Add Event Website Link** 
The "Confirm URL" link does not lead anywhere nor does it do anything

v 5.0.47
Android 13, Pixel 6

![Screenshot_20230816-085734](https://github.com/rokwire/illinois-app/assets/1044905/2680f9eb-5008-4922-816f-e27459a32a44)

![Screenshot_20230816-085628](https://github.com/rokwire/illinois-app/assets/1044905/fc176609-0faf-412d-8c81-699844dd4f05)

![Screenshot_20230817-100751](https://github.com/rokwire/illinois-app/assets/1044905/e3eca4ec-dca6-4691-88c8-f1e561def3e9)

**Comment by mihail-varbanov** [comment]
In version 5.0.51
1. Confirm URL link is added under Online Url text edit;
2. Confirm URL processing is improved to display status message in case the URL cannot be validated, to display progress indicator while preprocessing the URL link.
3. Before and now typing on Confirm URL adds schema (https, http) if it is omitted and updates the text edit.

**Comment by vburgett** [comment]
In 5.0.52, I can add "http://" and "confirm the url" for the "Add Event Website Link." However after tapping "Update Event," the "website" button on the event details panel still does not work. When going back into the event admin settings, I see that the edit did not stick and once again there is no "http://" in the "Add Event Website Link" field.

I am testing the event "World Photo Day" tomorrow 12am-12pm.

**Comment by mihail-varbanov** [comment]
I saw the problem and I cleaned up URL parsing, validation and processing in different aspects:

1. Report error if we fail to launch a URL, so that clicking on the Website button will not do nothing any more.

2. Require input of valid URL when creating a panel. In this line of thinking maybe we should rename "Confirm URL" link to "Validate URL".

3. Validate URLs before launching them from the detail panel. Your old event "World Photo Day" now opens the website link by adding https schema automatically.

This is available in version 5.0.53.
